app/core/ui/src/main/kotlin/com/hedvig/android/core/ui/insurance/DocumentRow.kt

```diff
@@ -0,0 +1,71 @@
+package com.hedvig.android.core.ui.insurance
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.hedvig.android.core.designsystem.newtheme.SquircleShape
+import com.hedvig.android.core.designsystem.preview.HedvigPreview
+import com.hedvig.android.core.designsystem.theme.HedvigTheme
+import com.hedvig.android.core.icons.Hedvig
+import com.hedvig.android.core.icons.hedvig.small.hedvig.ArrowNorthEast
+
+@Composable
+fun DocumentRow(
+  modifier: Modifier = Modifier,
+  title: String,
+  subTitle: String,
+  onClick: () -> Unit,
+) {
+  Row(
```

Comment:
So in Figma these icons seem to have a background color of f0f0f0.
In our design system, the color which maps to this in light mode is `surfaceVariant`. If you follow the references from here https://github.com/HedvigInsurance/android/blob/93966a0194e063771c25701f721ea26c42ab71b3/app/core/design-system/src/main/kotlin/com/hedvig/android/core/designsystem/material3/HedvigMaterial3Theme.kt#L82 you'll see.

I think you can wrap this entire thing with a `HedvigCard` and do something like
```
HedvigCard(
  onClick = onClick,
  colors = CardDefaults.outlinedCardColors(containerColor = MaterialTheme.colorScheme.surfaceVariant),
) {
 Box() ... this code here
 // And remove the background() modifier from it since HedvigCard now sets the color, same for onClick. Keep the inner Padding that you have though.
}
```

Oh and in this process, maybe consider changing the default shape of the HedvigCard to `shape: Shape = MaterialTheme.shapes.squircle,` like HedvigBigCard does. That Big card is an alternative version of the card which needs a minimum size which is bigger, hence the name :D That one is newer so that has squirce by default. Now that we're doing all this redesign it's okay to propagate such changes to random places, we're gonna take a look at them anyway.